cpp/example_code/elasticache/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of
+# the License is located at
+# http://aws.amazon.com/apache2.0/
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
Please reach out to this example https://github.com/awsdocs/aws-doc-sdk-examples/blob/master/cpp/example_code/ec2/CMakeLists.txt to find the latest style of CMake usage to integrate SDK